pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -286,6 +286,9 @@ type BatchSchedulerConfiguration struct {
 	// PriorityClassName stands for the name of k8s PriorityClass resource, it's being used in Volcano batch scheduler.
 	// +optional
 	PriorityClassName *string `json:"priorityClassName,omitempty"`
+	// MinResource stands for the resource list custom request for, it's being used in Volcano batch scheduler.
+	// +optional
+	MinResource apiv1.ResourceList `json:"minResource,omitempty"`
```

Comment:
Why it's named `MinResouce`? Does it define the lower limit of resource requests?

Reply:
Yes, this means the min resource request for gang scheduling

Reply:
The comment should make it clear that this defines the lower-bound limit, and how it affects the resource request to the scheduler.

Reply:
ping @Thor-wl

Reply:
Is there any progress?